sections/Deals/AcceptOrReject/AcceptOrRejectSection/AcceptOrRejectSection.tsx

```diff
@@ -186,11 +197,28 @@ const AcceptOrRejectDeal: FC<AcceptOrRejectDealProps> = ({
 					</>
 				),
 				subText: (
-					<TokenDisplay
-						symbol={underlyingDealTokenSymbol}
-						address={deal?.underlyingDealToken}
-						displayAddress={true}
-					/>
+					<FlexDivCol>
+						<StyledFlexDiv>
+							<TokenDisplay
+								symbol={underlyingDealTokenSymbol}
+								address={deal?.underlyingDealToken}
+								displayAddress={true}
+							/>
+						</StyledFlexDiv>
+						<StyledFlexDiv>
+							Total Supply: <br />
+							{underlyingTotalSupply ? formatNumber(underlyingTotalSupply) : '-'}
+						</StyledFlexDiv>
+						<StyledFlexDiv>
```

Comment:
lets remove the implied market cap section since it will be wrong often. and we need a tooltip around total supply explaining that if these are pre tokens for a pre traded token they might not have the correct total supply